src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const promise1 = new Promise((resolve) => {
+    const logo = document.querySelector('.logo');
+
+    logo.addEventListener('click', () => {
+      resolve();
+    });
+  });
+
+  // eslint-disable-next-line promise/param-names
+  const promise2 = new Promise((_, reject) => {
+    setTimeout(() => {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      reject();
```

Comment:
The rejection in `promise2` does not provide any error information. Consider passing an error message or object to `reject()` to make debugging easier.